server/controllers/gh.js

```diff
@@ -28,8 +28,11 @@ function getIssues(req, res) {
 }
 
 function getIssue(req, res) {
-    var issueRequestUrl = issuesRequestUrl + '/' + req.params.id;
+    _getIssueBody(req, res);
```

Comment:
Because function getIssue already exported for using as express middlware, there third argument - next().